src/specs/pages/page-home/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@ecl/specs-page-home",
+  "author": "European Commission",
+  "license": "EUPL-1.2",
+  "version": "4.6.3",
+  "description": "ECL Example page - Home Specs",
+  "publishConfig": {
```

Comment:
also here, i mean, more or less the result is the same, the package will not be installable by non members of the INNO organisation on npm, but for consistency i would use "private" explicitely as we did for all the other packages that we don't want to make public.